### PR TITLE
docs: Add logo & split README file

### PR DESCRIPTION
**Pull request by uptotec**
Hi,
It's October and Hacktoberfest just started and I thought this year I'll go throw the repositories that I used during this year in my projects and see if there is any help I can offer. react-native-background-actions helped me in two projects this year as I found it a pretty good solution for running background tasks in RN after a long-term searching for a way to implement such a feature.

This pull request contains two additions to the documentations:
1- a simple logo for the repository as logos gives a high quality impression.
2- splitting the README file to multiple files with links to each file to help keeping things organize. too many information and code snippets in one long file causes the reader to lost track of what he should do to start using the package. splitting the README file to INSTALL and USAGE files will help solving this problem.

thanks for this package and hope you'll find this PR useful. 😀

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,238 +1,34 @@
...
+    <br></br>
+    <img src="https://github.com/Rapsssito/react-native-background-actions/workflows/Release/badge.svg" />
+    <img src="https://img.shields.io/npm/dw/react-native-background-actions" />
+    <img src="https://img.shields.io/npm/v/react-native-background-actions?color=gr&label=npm%20version" />
+</p>
 
 React Native background service library for running **background tasks forever in Android & iOS**. Schedule a background job that will run your JavaScript when your app is in the background or foreground.
 
-### WARNING
+### <span style="color:Red">WARNING</span>
```
Comment (Rapsssito) [inline README.md:11]: This style is not working for me. Should we remove it?

**Review by Rapsssito — CHANGES_REQUESTED** [review]

**Comment by Rapsssito** [comment]
@uptotec, thanks for the great PR, all the documentation looks very nice.

File: README.md (hunk 1 above)
Comment (uptotec) [reply README.md:11]: Yes, It doesn't work in GitHub and I forgot to remove it.

**Comment by Rapsssito** [comment]
LGTM!

**Pull request merged**